frontend/src/components/homepage/sections/HomepageExperimental.tsx

```diff
@@ -26,8 +26,8 @@ const ExperimentalContent = () => {
           {t("tryLink")}
         </Button>
       </Link>
-      <h3>{t("cantDoHeader")}</h3>
-      <p className="font-sans-md line-height-sans-4">{t("cantDoParagraph")}</p>
+      <h4>{t("cantDoHeader")}</h4>
+      <p>{t("cantDoParagraph")}</p>
       <Grid row className="padding-y-2" gap="md">
```

Comment:
- Remove unnecessary typography classes
- Use lower heading level so they're nested appropriately under the section heading